generative/maisi/scripts/sample.py

```diff
@@ -0,0 +1,747 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from datetime import datetime
+import json
+
+
+import monai
+import torch
+from monai.transforms import Compose, SaveImage
+from monai.data import MetaTensor
+from monai.inferers import sliding_window_inference
+from monai.utils import set_determinism
+from tqdm import tqdm
+from generative.metrics import FIDMetric
+from generative.inferers import LatentDiffusionInferer
+from utils import binarize_labels, MapLabelValue, general_mask_generation_post_process, get_body_region_index_from_mask
+from find_masks import find_masks
+from augmentation import augmentation
+
+
+class ReconModel(torch.nn.Module):
```

Comment:
Will this one be moved into the core?

Reply:
Maybe we can plan it later? Or do you want it to be moved now?